### ISSUE TITLE
`groupsocketlisten` should print timestamp

### ISSUE THREAD

**Issue by RichiH** [Makki1/knxd#47]
It's quite annoying that I don't see when something went through the bus.

**Comment by Makki1** [comment]
I fully agree, created vbusmonitor1time (after the same discussion - which I lost) quite some years ago: https://github.com/Makki1/knxd/blob/master/src/examples/vbusmonitor1time.c

We should just not break existing things, there are several things that parse "groupsocketlisten" directly,
so I'd suggest
1) advance/add groupsocketlistenT to the examples (easy)
2) add a flag to the API (much better, but more complicated)
(every packet should have a timestamp IMHO)

And in any case, I'd vote for an ISO8601 timestamp.

**Comment by RichiH** [comment]
For now, the hackish, but quick, way would be to add `groupsocketlistenT` -- it's ugly as hell, but it does what it needs to do, would work quickly, and would be removed in favour of `knx monitor` or whatever, anyway.

The colons in `2015-02-18T04:57:56Z` suck if you ever redirect them to a filename, but else they are fine.

**Comment by Makki1** [comment]
Go ahead, I'd suggest some minor changes: groupsocketlisten was already way too long, groupsocketlistenT is even worse with camel-case (yes, I suggested it.. but meant this just as an example)

knxd-g ?

For a clean namespace, in fact is no "/example" but real usage, so lets create a "clients" folder (?)
